README.md

```diff
@@ -11,4 +11,5 @@ To facilitate easy navigation and access to the resources within this repository
 
 1. [**Scripts**](/scripts/README.md): Within this folder, you will find a collection of various scripts that have been developed to automate tasks and streamline processes. These scripts have proven to be valuable tools, enhancing productivity in various workflows.
 2. [**GitHub Actions**](/github-actions/README.md):  This folder contains a selection of GitHub Actions that have been designed to optimize workflows. These actions offer a reusable and configurable approach for achieving efficiency and consistency in your development practices.
-3. [**Terraform**](/terraform/README.md): In this folder, you will discover a comprehensive set of Terraform modules created to address common use cases. These modules helps you to rapidly provision and manage infrastructure resources with ease, leveraging the power and flexibility of Terraform.
\ No newline at end of file
+3. [**Terraform**](/terraform/README.md): In this folder, you will discover a comprehensive set of Terraform modules created to address common use cases. These modules helps you to rapidly provision and manage infrastructure resources with ease, leveraging the power and flexibility of Terraform.
+4. [**Ansible**](/ansible/README.md): In this folder, you will discover a comprehensive set of ansible roles to create/manage users. This role helps you to easily manage users in different platforms like AWS, Linux, Jenkins, MySQL and Postgres.
```

Comment:
In this folder, you will discover a comprehensive set of Ansible roles to run many common scenarios.

Reply:
Updated as mentioned.